src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -66,7 +66,7 @@ class QuantumMachinesClusterSettings(Instrument.InstrumentSettings):
         controllers: list[dict[str, Any]]
         elements: list[dict[str, Any]]
 
-        def to_qua_config(self) -> Dict[str, Any]:
+        def to_qua_config(self) -> DictQuaConfig:
```

Comment:
Aren't we missing the `oscillators` key, for it to be a `DictQuaConfig`?

Reply:
I like the change though! 

Cool concretization 👍

Reply:
Seems like we are not using the `oscillators` key anywhere, so that why they are not there! Resolving 👌